2022-02-14 23:58:04,192 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Loading settings from /N/u/soodn/Carbonate/.config/wandb/settings
2022-02-14 23:58:04,198 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Loading settings from /geode2/home/u090/soodn/Carbonate/ccf-research-kaggle-2021/models/5-Deepflash2/wandb/settings
2022-02-14 23:58:04,199 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-02-14 23:58:04,200 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'models/5-Deepflash2/Deepflash2-Train-Colon.py', 'program': '/geode2/home/u090/soodn/Carbonate/ccf-research-kaggle-2021/models/5-Deepflash2/Deepflash2-Train-Colon.py'}
2022-02-14 23:58:04,202 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-14 23:58:04,203 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-14 23:58:04,203 INFO    MainThread:36798 [wandb_init.py:_log_setup():386] Logging user logs to /geode2/home/u090/soodn/Carbonate/ccf-research-kaggle-2021/models/5-Deepflash2/wandb/run-20220214_235804-jqwqax1r/logs/debug.log
2022-02-14 23:58:04,204 INFO    MainThread:36798 [wandb_init.py:_log_setup():387] Logging internal logs to /geode2/home/u090/soodn/Carbonate/ccf-research-kaggle-2021/models/5-Deepflash2/wandb/run-20220214_235804-jqwqax1r/logs/debug-internal.log
2022-02-14 23:58:04,205 INFO    MainThread:36798 [wandb_init.py:init():420] calling init triggers
2022-02-14 23:58:04,206 INFO    MainThread:36798 [wandb_init.py:init():425] wandb.init called with sweep_config: {}
config: {}
2022-02-14 23:58:04,206 INFO    MainThread:36798 [wandb_init.py:init():441] re-initializing run, found existing run on stack: 35jvy3mw
2022-02-14 23:58:04,207 INFO    MainThread:36798 [wandb_run.py:finish():1468] finishing run soodn/bricknet/35jvy3mw
2022-02-14 23:58:04,208 INFO    MainThread:36798 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-02-14 23:58:04,213 INFO    MainThread:36798 [wandb_run.py:_restore():1769] restore
2022-02-14 23:58:06,793 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 904
  total_bytes: 904
}

2022-02-14 23:58:06,957 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 904
  total_bytes: 904
}

2022-02-14 23:58:07,293 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 904
  total_bytes: 19323
}

2022-02-14 23:58:07,404 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4709
  total_bytes: 19323
}

2022-02-14 23:58:07,508 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19323
  total_bytes: 19323
}

2022-02-14 23:58:07,612 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19323
  total_bytes: 19323
}

2022-02-14 23:58:07,715 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19323
  total_bytes: 19323
}

2022-02-14 23:58:07,819 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19323
  total_bytes: 19323
}

2022-02-14 23:58:08,286 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19323
  total_bytes: 19323
}

2022-02-14 23:58:08,487 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19323
  total_bytes: 19323
}
local_info {
}

2022-02-14 23:58:09,821 INFO    MainThread:36798 [wandb_run.py:_append_history():2144] rendering history
2022-02-14 23:58:09,822 INFO    MainThread:36798 [wandb_run.py:_append_summary():2102] rendering summary
2022-02-14 23:58:09,824 INFO    MainThread:36798 [wandb_run.py:_append_files():2194] logging synced files
2022-02-14 23:58:09,831 INFO    MainThread:36798 [wandb_init.py:init():471] starting backend
2022-02-14 23:58:09,831 INFO    MainThread:36798 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-14 23:58:09,836 INFO    MainThread:36798 [backend.py:ensure_launched():219] starting backend process...
2022-02-14 23:58:09,921 INFO    MainThread:36798 [backend.py:ensure_launched():224] started backend process with pid: 6931
2022-02-14 23:58:09,924 INFO    MainThread:36798 [wandb_init.py:init():480] backend started and connected
2022-02-14 23:58:09,936 INFO    MainThread:36798 [wandb_init.py:init():550] updated telemetry
2022-02-14 23:58:10,258 INFO    MainThread:36798 [wandb_init.py:init():581] communicating current version
2022-02-14 23:58:15,202 INFO    MainThread:36798 [wandb_init.py:init():586] got version response 
2022-02-14 23:58:15,203 INFO    MainThread:36798 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-02-14 23:58:15,598 INFO    MainThread:36798 [wandb_init.py:init():624] starting run threads in backend
2022-02-14 23:58:20,609 INFO    MainThread:36798 [wandb_run.py:_console_start():1827] atexit reg
2022-02-14 23:58:20,611 INFO    MainThread:36798 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.REDIRECT
2022-02-14 23:58:20,611 INFO    MainThread:36798 [wandb_run.py:_redirect():1706] Redirecting console.
2022-02-14 23:58:20,613 INFO    MainThread:36798 [wandb_run.py:_redirect():1762] Redirects installed.
2022-02-14 23:58:20,613 INFO    MainThread:36798 [wandb_init.py:init():651] run started, returning control to user process
2022-02-14 23:58:25,322 INFO    MainThread:36798 [wandb_run.py:_config_callback():966] config_cb None None {'Learner': {'loss_func': 'JointLoss(\n  (first): WeightedLoss(\n    (loss): TorchLoss(\n      (loss): DiceLoss()\n    )\n  )\n  (second): WeightedLoss()\n)', 'opt_func': 'fastai.optimizer.ranger', 'lr': 0.001, 'splitter': 'fastai.torch_core.trainable_params', 'metrics': ['<hubmap_loss_metrics.Dice object at 0x2b9a500cd310>', '<hubmap_loss_metrics.Iou object at 0x2b9a500cd4c0>', '<hubmap_loss_metrics.Recall object at 0x2b9a500cd520>', '<hubmap_loss_metrics.Precision object at 0x2b9a500cd580>'], 'path': '.', 'model_dir': 'models', 'wd': 0.0001, 'wd_bn_bias': False, 'train_bn': True, 'moms': [0.95, 0.85, 0.95], '_name': '<fastai.learner.Learner object at 0x2b9a5a1c1070>'}, 'TrainEvalCallback': True, 'Recorder': {'add_time': True, 'train_metrics': False, 'valid_metrics': True}, 'ProgressCallback': True, 'SaveModelCallback': {'fname': 'model', 'every_epoch': False, 'at_end': False, 'with_opt': False}, 'ElasticDeformCallback': True, 'WandbCallback': {'log': 'gradients', 'log_preds': False, 'log_model': False, 'log_dataset': False, 'dataset_name': None, 'valid_dl': None, 'n_preds': 36, 'seed': 12345, 'reorder': True}, 'MixedPrecision': True, 'ParamScheduler': True, 'n_inp': 1, 'input 1 dim 1': 32, 'input 1 dim 2': 3, 'input 1 dim 3': 512, 'input 1 dim 4': 512, 'batch size': 32, 'batch per epoch': 18, 'model parameters': 10046436, 'device': 'cuda', 'frozen': False, 'frozen idx': 0}
2022-02-15 00:09:24,497 INFO    MainThread:36798 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-02-15 00:09:24,507 INFO    MainThread:36798 [wandb_run.py:_restore():1769] restore
2022-02-15 00:09:27,290 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 904
  total_bytes: 904
}

2022-02-15 00:09:27,412 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 904
  total_bytes: 904
}

2022-02-15 00:09:28,009 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 904
  total_bytes: 12337
}

2022-02-15 00:09:28,111 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 904
  total_bytes: 12337
}

2022-02-15 00:09:28,215 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12337
  total_bytes: 12337
}

2022-02-15 00:09:28,319 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12337
  total_bytes: 12337
}

2022-02-15 00:09:28,422 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12337
  total_bytes: 12337
}

2022-02-15 00:09:28,526 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12337
  total_bytes: 12337
}

2022-02-15 00:09:28,825 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12337
  total_bytes: 12337
}

2022-02-15 00:09:29,023 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12337
  total_bytes: 12337
}
local_info {
}

2022-02-15 00:09:30,337 INFO    MainThread:36798 [wandb_run.py:_append_history():2144] rendering history
2022-02-15 00:09:30,339 INFO    MainThread:36798 [wandb_run.py:_append_summary():2102] rendering summary
2022-02-15 00:09:30,339 INFO    MainThread:36798 [wandb_run.py:_append_files():2194] logging synced files
